app/js/lib/view_handler/attribute_value_handler.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+	_ = require( 'underscore' ),
+
+	AttributeValueHandler = {
+		'post-code': null,
+		city: null,
+		email: null,
+
+		update: function ( elements ) {
+			var self = this;
+			_.each( elements, function ( attributes, elementId ) {
+				self.setAttributesForElement( self[ elementId ], attributes );
```

Comment:
If you change the constructor to accept a jQuery object, you can replace `self[ elementId ]` with something like 

    self.elements.find( elementId );

It'll make the unit test a bit more complicated, but you'll manage.